.github/workflows/openapi.yml

```diff
@@ -0,0 +1,57 @@
+name: openapi
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deref_json:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Getsentry Token
+      id: getsentry
+      uses: getsentry/action-github-app-token@v1
+      with:
+        app_id: ${{ secrets.SENTRY_INTERNAL_APP_ID }}
+        private_key: ${{ secrets.SENTRY_INTERNAL_APP_PRIVATE_KEY }}
+
+    - name: Checkout getsentry/sentry
+      uses: actions/checkout@v2
+      with:
+        path: sentry
+
+    - name: Checkout getsentry/sentry-api-schema
+      uses: actions/checkout@v2
+      with:
+        ref: 'main'
+        repository: getsentry/sentry-api-schema
+        path: sentry-api-schema
+        token: ${{ steps.getsentry.outputs.token }}
+
+    # Install/setup node
+    - uses: volta-cli/action@v1
+
+    - name: Build OpenAPI Derefed JSON
+      run: |
+        cd sentry
+        yarn install --frozen-lockfile
+        yarn run build-derefed-docs sentry/api-docs/openapi-derefed.json
+
+    - name: Copy artifact into getsentry/sentry-api-schema
+      run: |
+        ls
+        cp sentry/api-docs/openapi-derefed.json sentry-api-schema
+
+    - name: Git Commit & Push
+      run: |
+        cd sentry-api-schema
+        if [ -n "$(git status --porcelain)" ]; then
+          git config user.name openapi-getsentry-bot
```

Comment:
This is a pretty common thing to do in actions. You might want to just invoke an existing action:
https://github.com/marketplace/actions/add-commit
https://github.com/marketplace/actions/git-commit-and-push

Reply:
@mgaeta yeah but for something pretty simple like this, I don't think its needed.

Reply:
GitHub Actions are perfect for simple things, it's less code to maintain:
```
- name: Git Commit & Push
- uses: stefanzweifel/git-auto-commit-action@v4
  with:
    branch: main
    commit_message: Generated
    commit_user_email: bot@getsentry.com
    commit_user_name: openapi-getsentry-bot
```